docs/UserGuide.md

```diff
@@ -8,28 +8,27 @@ optimized for use via a Command Line Interface** (CLI) while still having the be
 If you can type fast, PROperty can get your contact and property management tasks done faster than traditional GUI apps.
 
 PROperty is useful for property agents because it saves their time by allowing easy tracking of contacts,
-and easily filtering them according to tailor-made categories relevant to property agents in Singapore. It taps on the
-fact that it is a lot less complicated to operate than its competitors such as Microsoft Excel while being faster to
-operate than graphical user interfaces commonly found in smartphones.
+and easily filtering them according to tailor-made categories relevant to property agents in Singapore. It is much
+simpler to use while being even more functional than alternatives on the market.
 
 ## Table of Contents
 * [Quick start](#quick-start)
 * [Features](#features)
     * [Adding a client: `add`](#adding-a-client-add)
+    * [Editing a client : `edit`](#editing-a-client--edit)    
+    * [Deleting a client : `delete`](#deleting-a-client--delete)
     * [Listing all clients : `list`](#listing-all-clients--list)
-    * [Editing a client : `edit`](#editing-a-client--edit)
+    * [Sorting all clients : `sort`](#sorting-all-clients--sort)
+    * [Showing property listings of a client : `show`](#showing-property-listings-of-a-client--show)
     * [Locating clients by name: `find`](#locating-clients-by-name-find)
     * [Locating clients by tag: `findtag`](#locating-clients-by-tag-findtag)
-    * [Deleting a client : `delete`](#deleting-a-client--delete)
     * [Clearing all entries : `clear`](#clearing-all-entries--clear)
-    * [Exiting the program : `exit`](#exiting-the-program--exit)
-    * [Quick reference help: `help`](#quick-reference-help-help)
     * [Managing Remarks : `remark`](#managing-remarks--remark)
-    * [Showing property listings of a client : `show`](#showing-property-listings-of-a-client--show)
     * [Adding a property listing : `listing add`](#adding-a-property-listing--listing-add)
     * [Deleting a property listing : `listing delete`](#deleting-a-property-listing--listing-delete)
     * [Exporting your contacts : `export`](#exporting-your-contacts--export)
-    * [Sorting all clients : `sort`](#sorting-all-clients--sort)
+    * [Exiting the program : `exit`](#exiting-the-program--exit)
+    * [Quick reference help: `help`](#quick-reference-help-help)
```

Comment:
Do you think we should move the indent to the same level as `Feature` and `Saving the data` instead? Or maybe we can rename it to something other than `Quick reference help`? I felt that the prof had an issue with us putting something labeled as `Quick reference` under features

Reply:
I feel like once we move the help to the end of features it's ok. I think what prof meant that it should not be in the middle of features. She was not really specific about whether it should be at the end of features or completely outside of features. Good comment though we should discuss with the rest of the team.